explainer.md

```diff
@@ -118,6 +136,12 @@ dictionary ItemDetails {
   DOMString description;
   PaymentCurrencyAmount price;
 };
+
+dictionary PurchaseDetails {
+  DOMString itemId;
+  DOMString purchaseToken;
+  boolean isAcknowledged;
```

Comment:
I think we want purchaseState rather than just isAcknowledged, e.g. for pending purchases.

Reply:
Do we want both? They seem to overlap. I'll put in both for now.